lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -565,11 +602,13 @@ class ReportUIFeatures {
     // Category order matches gauge order in sticky header.
     const gaugeWrapperEls = this.stickyHeaderEl.querySelectorAll('.lh-gauge__wrapper');
     const gaugeToHighlight = gaugeWrapperEls[highlightIndex];
-    const offset = gaugeToHighlight.getBoundingClientRect().left + 'px';
+    // This is normally 0 - but in DevTools, it's not, since the entire report could be docked.
```

Comment:
weird, apparently devtools covers the whole window when docked but has a weird ghost element taking up the other section of the page where the content goes that's not selectable with inspect, etc

Reply:
heh, yeah one of the 🤯moments for me was learning that with devtools the entire page is really devtools and we're just sticking this other renderer inside it that happens to take up most of the space

Reply:
> 🤯

lol how is that in unicode but a lighthouse isn't

also (at least on a mac), that looks a lot gorier small than it does big and you realize that's a mushroom cloud coming out

Reply:
can we poke someone about the two pending Lighthouse unicode proposals? :)